src/main/java/bio/terra/cda/app/util/CDAUtils.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.cda.app.util;
+
+import java.util.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CDAUtils {
+  private static final Logger logger = LoggerFactory.getLogger(CDAUtils.class);
+
+  /** Split the TABLE.ColumnName into component parts */
+  public static Map<String, String> parseTableName(String tableColumnString) {
+    List<String> parts = new LinkedList<String>(Arrays.asList(tableColumnString.split("\\.")));
+    int columnIndex = parts.size() - 1;
+    String column = parts.get(columnIndex); // last bit is the column
+    parts.remove(columnIndex); // remove the column part
+    String tableName = String.join(".", parts);
+
+    Map<String, String> tableMap = new HashMap<String, String>();
+    tableMap.put("columnName", column);
+    tableMap.put("tableName", tableName);
+
+    return tableMap;
```

Comment:
With JDK 11 this can be done using `Map.of()`
```suggestion
    return Map.of("columnName", column, "tableName", tableName);
```